src/robusta/core/playbooks/internal/crds.py

```
@@ -0,0 +1,310 @@
+        file_block = FileBlock(
+            filename=filename,
+            contents=output.encode()
+        )
+
+        finding.add_enrichment([file_block])
+        event.add_finding(finding)
+
+    except Exception:
+        msg = f"Error running kubectl_describe for {params}"
+        logging.exception(msg)
+        raise ActionException(ErrorCodes.ACTION_UNEXPECTED_ERROR, msg)
```

Ruff (0.14.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)